client/src/components/products/Card.js

```diff
@@ -0,0 +1,19 @@
+import { Link } from "react-router-dom";
+import { CardContainer } from "../home/FeaturedProducts";
+
+function Card({ product }) {
+  return (
+    <CardContainer>
+      <div className="prod-image">
+        <img src={product.imageUrl} alt="" />
```

Comment:
alt here too

Reply:
think every product object should have an alt tag in it so i can render it easily.